tests/test_fuzz.py

```
@@ -0,0 +1,110 @@
+try:
+	import sys
+	if sys.__name__ is None:  # pragma: no branch
+		raise ModuleNotFoundError("[CWE-440] OMG! we could not import sys! ABORT. ABORT.") from None
+except Exception as err:  # pragma: no branch
+	raise ImportError(err) from err
+
+
+try:
+	try:
+		import context
+	except Exception as ImportErr:  # pragma: no branch
```

Ruff

Local variable `ImportErr` is assigned to but never used

Remove assignment to unused variable `ImportErr`

(F841)